pkg/resources/stream_acceptance_test.go

```diff
@@ -52,8 +52,8 @@ func TestAcc_StreamCreateOnStage(t *testing.T) {
 }
 
 func TestAcc_Stream(t *testing.T) {
-	env := os.Getenv("STREAM_TEST")
-	if env == "" {
+	env := os.Getenv("SKIP_STREAM_TEST")
```

Comment:
I've checked and couldn't see where this env is set, maybe we could remove it when we're not using it now ?